tests/unit/array/storage/elastic/test_add.py

```diff
@@ -0,0 +1,43 @@
+from docarray import Document, DocumentArray
+
+
+def test_add_ignore_existing_doc_id(start_storage):
+    elastic_doc = DocumentArray(
+        storage='elasticsearch',
+        config={
+            'n_dim': 3,
+            'columns': [('price', 'int')],
+            'distance': 'l2_norm',
+            'index_name': 'test_add',
```

Comment:
can you perhaps randomize the index name or give the index name the exact same name as the test function name

Reply:
oh sure will do